doc/API.rst.txt

```diff
@@ -143,6 +143,7 @@ change is made.
 
 * `scipy.stats`
 
+  - `scipy.stats.contingency`
   - ``scipy.stats.distributions``
```

Comment:
Just seeing this. Is it actually normal that we don't link here? Seems like it's the only exception.

Reply:
It's because `distributions` is one of those "accidentally kinda public" submodules, and therefore doesn't have a dedicated page - so the link wouldn't go anywhere. We need to fix this at some point. Unfortunately pretty much everything, with a few exceptions like `rv_frozen`, is duplicate with the main `stats` namespace.

Reply:
I see, thanks!